charts/aws-efs-csi-driver/templates/controller-serviceaccount.yaml

```diff
@@ -40,6 +40,9 @@ rules:
   - apiGroups: ["coordination.k8s.io"]
     resources: ["leases"]
     verbs: ["get", "watch", "list", "delete", "update", "create"]
+  - apiGroups: [ "" ]
+    resources: [ "secrets" ]
+    verbs: [ "get", "watch", "list" ]
```

Comment:
What's the effect of these configurations? Why the apiGroups are empty?

Reply:
Empty apiGroups indicates it applies to core k8s APIs. 

This RBAC rule authorizes a service account associated with controller to read k8s [secret](https://kubernetes.io/docs/concepts/configuration/secret/) objects